### PR TITLE
add program files

### PR DESCRIPTION
**Pull request by yuliichka**

### CODE REVIEW

**Comment by roman-stozhuk** [comment]
**добре гарно**

**Comment by dmytro-maherovskyi-lpnu** [comment]
> **добре гарно**

Зараз перевіримо =)

File: people_data.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import re
+from datetime import datetime
+
+class PeopleData:
+    _id_counter = 1
```
Comment (dmytro-maherovskyi-lpnu) [inline people_data.py:5]: We won't be required to generate ID in code in the future as primary keys are set automatically. But let it be for now

File: people_data.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+    @property
+    def id(self):
+        return self._id
+
+    @staticmethod
+    def validate_name(name):
+        if name.isalpha():
+            return name
+        raise ValueError("Ім'я та прізвище повинні складатися тільки з букв.")
```
Comment (dmytro-maherovskyi-lpnu) [inline people_data.py:29]: If you check Elon Musk's daughter name - that's not always a case

File: people_data.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+    @staticmethod
+    def validate_email(email):
+        email_regex = r'^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$'
```
Comment (dmytro-maherovskyi-lpnu) [inline people_data.py:33]: Nice!

File: professor.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+        self._courses = []
+
+    def add_course(self, course_name):
+        self._courses.append(course_name)
+        return f"Курс {course_name} додано."
+
+    def get_courses(self):
+        return f"Курси, які викладає: {', '.join(self._courses)}" if self._courses else "Немає курсів для викладання."
+
+    def get_info(self):
```
Comment (dmytro-maherovskyi-lpnu) [inline professor.py]: Can be made as get-only property

File: student.py [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+        current_year = datetime.now().year
+        if isinstance(year, int) and 1900 <= year <= current_year:
+            return year
+        raise ValueError("Рік вступу повинен бути коректним числом у межах 1900 і поточного року.")
+
+    def enroll(self, course):
+        self._enrollments.append(course)
+
+    def drop_course(self, course_name):
+        self._enrollments = [e for e in self._enrollments if e != course_name]
```
Comment (dmytro-maherovskyi-lpnu) [inline student.py]: To avoid this check - use dictionary with course_name as a key

**Review by dmytro-maherovskyi-lpnu — APPROVED** [review]

File: people_data.py (hunk 1 above)
Comment (yuliichka) [reply people_data.py:5]: Ok, got it

File: professor.py (hunk 4 above)
Comment (yuliichka) [reply professor.py]: Done

**Review by dmytro-maherovskyi-lpnu — APPROVED** [review]

**Pull request merged**